index.html

```diff
@@ -30,37 +33,194 @@ <h1 class="title">I’m Atif<br>
             <p class="description">I’m a software developer! I can help you build a product , feature or website Look
                 through some of my work and experience! If you like what you see and have a project you need coded,
                 don’t hestiate to contact me.</p>
-                <div class="icon">
-                    <p class="text">LET’S CONNECT</p>
-                    <ul>
-                        <li class="twitter">
-                            <a href="https://twitter.com/Atifkha80520602">
-                                <img src="assets/twitter icon.svg" alt="twitter icon">
-                              </a>
+            <div class="icon">
+                <p class="text">LET’S CONNECT</p>
+                <ul class="social-icon">
+                    <li class="twitter">
+                        <a href="https://twitter.com/Atifkha80520602">
+                            <img src="assets/twitter icon.svg" alt="twitter icon">
+                        </a>
+                    </li>
+                    <li class="linkedin">
+                        <a href="https://www.linkedin.com/in/atif-zada-585693180/">
+                            <img src="assets/Linkedin icon.svg" alt="linkedin icon">
+                        </a>
+                    </li>
+                    <li class="github">
+                        <a href="https://github.com/Atifzada">
+                            <img src="assets/m icon.svg" alt="github icon">
+                        </a>
+                    </li>
+                    <li class="med">
+                        <a href="#">
+                            <img src="assets/github icon.svg" alt="medium icon">
+                        </a>
+                    </li>
+                    <li class="angellist">
+                        <a href="#">
+                            <img src="assets/angellist icon.svg" alt="medium icon">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section class="workcards">
+        <div class="card">
+            <div class="protfolio-img">
+                <img src="./assets/works img/work1.svg" alt="card1" />
+            </div>
+            <div class="works-details">
+                <h1 class="comp-title">Tonic</h1>
+                <ul class="work-title">
+                    <li class="client">CANOPY</li>
+                    <li><img src="assets/works img/Counter.svg" alt="dot" /></li>
+                    <li class="dim">Back End Dev</li>
+                    <li><img src="assets/works img/Counter.svg" alt="dot" /></li>
+                    <li class="dim">2015</li>
+                </ul>
+                <div class="project-desc">
+                    <span>A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.</span>
```

Comment:
- [ ] As this ```("A daily section ...") ``` is a paragraph, I think it would be better to use a ```<p>``` tag instead of ```<span>```, The purpose of the "p" tag is to structure the content of a web page by organizing text into logical groups. 😎